frontend/server/k8s-helper.ts

```diff
@@ -127,3 +131,30 @@ export function getPodLogs(podName: string): Promise<string> {
       (error: any) => { throw new Error(JSON.stringify(error.body)); }
     );
 }
+
+export async function newTensorboardInstance(logdir: string): Promise<void> {
+  if (!k8sV1CustomObjectClient) {
+    throw new Error('Cannot access kubernetes Custom Object API');
+  }
+  // TODO: take the configuration below to a separate file
+  const group = 'kubeflow.org';
+  const version = 'v1beta1';
+  const namespace = 'kubeflow';
+  const plural = 'tensorboard';
+  const body = {
+    apiVersion: 'kubeflow.org/v1beta1',
```

Comment:
Any reason not to use the above constants here? (and below)

Reply:
A quick question: if I pass "group" and "version" parameters to the method createNamespacedCustomObject, do I still need to specify "apiVersion" field in "body" parameter? Or I can just leave "apiVersion" field unset in "body"?

Reply:
Done with using the constants